src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_PaymentOptionsScreen.tsx

```diff
@@ -49,10 +57,40 @@ export const Profile_PaymentOptionsScreen = ({}: PaymentOptionsProps) => {
   }
 
   useEffect(() => {
-    if (isFocused) {
-      refreshCards();
-    }
-  }, [isFocused]);
+    refreshCards();
```

Comment:
isFocused is removed to prevent reloading the cards when you return from the add card process, and corresponding race conditions in setting `isFocused`. The only (minor) consequence is that if you navigate to this page first, then purchase a ticket, and finally navigate back here again, the card you just saved won't appear until you pull down to refresh. That's considered acceptable.

Reply:
Now there are two useEffects with empty dependency arrays. Any particular reason why not just one, or can refreshCards(); be moved inside the other one?